src/app/core/components/arrow-text/arrow-text.component.html

```diff
@@ -1,13 +1,23 @@
 <button class="invisible-button" [disabled]="disabled">
   <div class="arrow-container" appSwap [isSwapped]="isRight">
-    <div
+    <!-- <div
       class="text-container"
       [ngClass]="{ 'text-right': isRight, 'text-left': !isRight }"
     >
       <span class="text-main" translate>{{
         isRight ? 'Next Step' : 'Previous Step'
       }}</span>
       <span class="text-sub">{{ mainText || '' }}</span>
+    </div> -->
```

Comment:
Can you remove unused code ? you can use git lens to see back what's deleted later.